pkgs/by-name/co/cosmic-comp/package.nix

```diff
@@ -41,7 +45,19 @@ rustPlatform.buildRustPackage {
   separateDebugInfo = true;
 
   nativeBuildInputs = [ makeBinaryWrapper pkg-config ];
-  buildInputs = [ libglvnd libinput libxkbcommon mesa seatd udev wayland ];
+  buildInputs =
+    [
+      libglvnd
+      libinput
+      libxkbcommon
+      mesa
+      seatd
+      udev
+      wayland
+    ] ++ lib.optional useSystemd systemd;
```

Comment:
What does this do?  We didn't have an explicit systemd dependency before.

Reply:
Ah, I see, it's used for readiness notification and some other things.  Makes sense.